hadoop-hdds/server-scm/src/test/java/org/apache/hadoop/hdds/scm/ha/TestReplicationAnnotation.java

```diff
@@ -88,8 +88,7 @@ public void testReplicateAnnotationBasic() throws Throwable {
 
     try {
       proxy.addContainer(HddsProtos.ContainerInfoProto.getDefaultInstance());
-      // Should have seen a IOException.
-      Assert.fail();
+      Assert.fail("Cannot reach here: should have seen a IOException");
```

Comment:
Have this line of change due to https://github.com/apache/ozone/pull/1650#discussion_r551137844